scripts/install-pytorch.sh

```diff
@@ -8,25 +8,25 @@ BUILD_LATEST=false
 FORCE_REINSTALL=false
 PYTORCH_CURRENT_COMMIT=""
 VENV=false
+NO_CLEAN=false
```

Comment:
Let's use the opposite, it's easier to read:
```suggestion
CLEAN=true
```